uxarray/grid/geometry.py

```diff
@@ -19,21 +19,25 @@
 
 # General Helpers for Polygon Viz
 # ----------------------------------------------------------------------------------------------------------------------
-@njit
+# @njit
```

Comment:
Why's this?

Reply:
Good catch, i had it turned off for debugging (break points don't work well with numba enabled)